app/models/event_schema.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# An EventSchema establish a default config for each event so that each
+# external events will be processed based on the config.
+
+# For more information, see https://github.com/gbrlmrllo/chatops_builder_api/issues/11
+class EventSchema < ApplicationRecord
+  belongs_to :creator, class_name: "User", optional: false
+  belongs_to :app
+  # has_many :events # TODO
+  # has_many :integrations # TODO
+
+  validates :name, presence: true, uniqueness: { scope: :creator_id }
+
+  scope :schema_structure, -> { { data: [], recipient: [] } }
```

Comment:
Scope methods are generally used to define queries, it is confusing to see it and that it does not implement an ActiveRecord query, it is preferable to declare this as a class method.

```ruby 
def self.schema_structure
   { data: [], recipient: [] }
end
```

Should also add a spec for this method.